src/barkhaPJun20/FirstLastIndexOfMultiOccurence.java

```diff
@@ -0,0 +1,35 @@
+package barkhaPJun20;
+
+public class FirstLastIndexOfMultiOccurence {
+	
+	void findFirstLastIndex(String name) {
+		for(int outerIndex=0; outerIndex<name.length(); outerIndex++) {
```

Comment:
No need to give a variable name as outerIndex, index is fine.

Reply:
Done.
![image](https://user-images.githubusercontent.com/66830634/85757331-ff6d0400-b72c-11ea-9ed8-f029e1c6701f.png)